build.gradle

```diff
@@ -438,8 +438,21 @@ task setVersion << {
         releases.write(releases.getText().replaceAll("# add new version here", "# add new version here\n    releases/release-${to}"))
 
         // release-<versionr>.rst
-        file("digdag-docs/src/releases/release-${to}.rst").append("")
-        "git add digdag-docs/src/releases/release-${to}.rst".execute().waitFor()
+        String notePath = "digdag-docs/src/releases/release-${to}.rst"
+        file(notePath).append(
+                "Release ${to}\n" +
+                "==============\n" +
+                "\n" +
+                "...\n" +
+                "\n" +
+                "Release Date\n" +
+                "------------\n" +
+                "${new Date().format('yyyy-MM-dd')}\n" +
+                "\n" +
+                "Contributors\n" +
+                "------------\n")
+        ["bash", "-c", "git shortlog -s \$(git tag --sort=-creatordate | head -n 1)..HEAD | awk '{print \"* \" \$2}' >> ${notePath}"].execute().waitFor()
```

Comment:
> `$2`

If one of rows from `git shortlog` is `   1234   Foo Bar`, this only captures `Foo`?

Reply:
`awk '{$1 = ""; print "* " $0}'`
Like this?

Reply:
Good catch...! Sadayuki Furuhashi was becoming Sadayuki...